src/components/data-processing/ConfigureEmbedding/CalculationConfig.jsx

```diff
@@ -119,6 +118,11 @@ const CalculationConfig = (props) => {
     updateSettings(changes);
     setChangesOutstanding(false);
     dispatch(loadEmbedding(experimentId, embeddingMethod, true));
+    if (changedFilters?.current.size) {
+      // other steps are changed so we run the pipeline
+      changedFilters.current.add(FILTER_UUID);
+      onPipelineRun();
+    }
```

Comment:
Maybe this should be an `if/else`? If there are changes to the previous filters, `dispatch(loadEmbedding())` doesn't need to be called before running the pipeline.